opennms-full-assembly/src/assembly/core.xml

```diff
@@ -0,0 +1,16 @@
+<assembly
+    xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd"
+>
+  <id>core</id>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <componentDescriptors>
+    <componentDescriptor>src/assembly/components/adapters.xml</componentDescriptor>
+    <componentDescriptor>src/assembly/components/base-assembly.xml</componentDescriptor>
+    <componentDescriptor>src/assembly/components/osgi.xml</componentDescriptor>
+    <componentDescriptor>src/assembly/components/selenium-monitor.xml</componentDescriptor>
+    <componentDescriptor>src/assembly/components/webapps.xml</componentDescriptor>
+    <componentDescriptor>src/assembly/components/xml.xml</componentDescriptor>
+  </componentDescriptors>
```

Comment:
is the xml-contrib.xml file meant to be excluded from this list?

Reply:
from core? yes, it's "bonus content" :)